.vscode-test.js

```diff
@@ -5,6 +5,18 @@ export default defineConfig([
     files: "out/**/*.test.js",
     mocha: {
       ui: "bdd",
+      color: true,
+      timeout: 10_000,
```

Comment:
Default timeout is 2sec, which almost always fails if we have to await the `getTestExtensionContext()`